security_controls_scp/variables.tf

```
@@ -12,4 +12,19 @@ variable "region_lockdown" {
     "us-east-2",
     "us-west-1"
   ]
+}
+
+variable "AmiCreatorAccount" {
```

I'm thinking we can make this variable name [snake case](https://en.wikipedia.org/wiki/Snake_case), as is the de facto standard for Terraform. 
```suggestion
variable "ami_creator_account" {
```

---

Good Call, updated.